source/material_model/rheology/drucker_prager.cc

```diff
@@ -199,7 +202,8 @@ namespace aspect
                            "those corresponding to chemical compositions. "
                            "The extremely large default cohesion value (1e20 Pa) prevents the viscous stress from "
                            "exceeding the yield stress. Units: \\si{\\pascal}.");
-        prm.declare_entry ("Maximum yield stress", "1e12", Patterns::Double (0.),
+        prm.declare_entry ("Maximum yield stress", "1e12",
+                           Patterns::Anything(),
```

Comment:
is it worth mentioning in the description of this parameter here, that this can be specified by compositional field?